temporalio/sig/temporalio/priority.rbs

```diff
@@ -1,8 +1,10 @@
 module Temporalio
   class Priority
     attr_reader priority_key: Integer?
+    attr_reader fairness_key: String?
+    attr_reader weight: Float?
 
-    def initialize: (priority_key: Integer?) -> void
+    def initialize: (priority_key: Integer?, ?fairness_key: String?, ?weight: Float?) -> void
```

Comment:
Mentioned in other comment, but you'll need to write initialize if you want optional parameters (the default `initialize` of `Data` does not support default parameters, but it's easy to write yourself and call super)